### ISSUE TITLE
potential transform issue

### ISSUE THREAD

**Issue by hkaraoguz** [jnkl314/DeepLabV3FineTuning#5]
Hi,
In this part of the code https://github.com/jnkl314/DeepLabV3FineTuning/blob/14d535982f50580b5ba0fbfd307e46c2cb93219c/sources/dataloader.py#L35

the transforms are applied to both the image and label masks but I think the masks should not be normalized. There is an example implementation that avoids normalizing the masks: You can follow the thread here:

https://github.com/pytorch/vision/blob/3c81d474a2525b11356ca1dde299a9b1ab3715c6/references/segmentation/transforms.py#L85
https://discuss.pytorch.org/t/where-are-the-masks-unnormalized-for-segmentation-in-torchvision-train-file/48113
